gph/python/test/test_ripser.py

```diff
@@ -311,3 +311,10 @@ def test_gens_order_vertices_higher_dimension():
 
     assert len(gens_fin_dim2) == 1
     assert np.array_equal(gens_fin_dim2[0], np.array([1, 0, 5, 4]))
+
+
+def test_ph_maxdim_0():
+    """Regression test for issue #39, an issue was found when only computing
+    up to dimension 0. The test also compares the results of dimension 0"""
```

Comment:
I guess you originally meant to say "The test also compares the results of dimension 0" when `maxdim=1` and `maxdim=0`?

Reply:
That was what I had previously ..........
Let me remove this part

Reply:
... or add the extra check in the test, which would be good also!

Reply:
Let me add the check then ;)

Reply:
Done